docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  app:
+    image: aalekhpatel07/student-net:latest

```

Comment:
Maybe we should leave this as
`build: .`
So we can make changes to the Dockerfile if necessary and don't rely on the shared version.

Reply:
Sure! That sounds good.